test/unit/puppetlabs/services/jruby/jruby_puppet_testutils.clj

```diff
@@ -126,7 +128,8 @@ create-mock-pool-instance :- JRubyInstance
                   :master-log-dir log-dir
                   :use-legacy-auth-conf false})
                 (jruby-core/initialize-config {:ruby-load-path ruby-load-path
-                                               :gem-home gem-home}))
+                                               :gem-home gem-home
+                                               :environment-vars environment-vars}))
```

Comment:
I'd prefer to leave the `:environment-vars` out of the common test helpers.  The setting of `:FOO` and `:BAR` values seems specific to the new tests that you're creating so I would scope those settings to only be used in the new tests that you're adding for this PR.